### DISCUSSION TITLE
Error with Caracal Names

### DISCUSSION THREAD

**Discussion by jadeitemods** [AquUpd/CaracalMob#8] [Q&A] [answered]
Hello,

Firstly, I wanted to mention how much I love this mod. That being said, I have discovered what I think is a bug on the most recent 1.17.1 Fabric release. Below, I'll provide screenshots

![image](https://user-images.githubusercontent.com/91622694/135306808-c9cfbd92-757d-4bda-9153-54239b2951da.png)
![image](https://user-images.githubusercontent.com/91622694/135307168-6e02d04a-6c54-4c3a-bb62-7d898d1fd11c.png)
![image](https://user-images.githubusercontent.com/91622694/135307304-8288f886-a299-4d4a-99f7-c954625c8d3f.png)
![image](https://user-images.githubusercontent.com/91622694/135307514-edbd991d-bb12-49e1-b731-4838ce7cedd6.png)

Commander, Peaceful Commander, and Explosive all generate the hat specified here:
![image](https://user-images.githubusercontent.com/91622694/135307824-ac7e685b-ed44-4749-b31b-841b83c4c756.png)

I don't know if I'm just behind or this is a bug of some sort of bug. And I'm not sure if anyone has reported this before. Is there a possibility that another mod is doing this?

I'll be easier to reach via response over at my email, Jadeite248@gmail.com. Thanks!

**Comment by Ivan-Khar** [comment]
Maybe you've got older version of mod?

**Comment by jadeitemods** [comment]
I'm using the version "caracal-1.17-1.3.11".

**Reply by Ivan-Khar** [reply]
Latest version of this mod is 1.3.2

**Reply by Ivan-Khar** [reply]
If you need to use this specific version you can check older wiki page

**Comment by jadeitemods** [comment]
I have the mod installed on my server and mod folder both, so it can't be
an issue of client side vs server side.

**Comment by jadeitemods** [comment]
Ah. I'll try that and let you know if it worked.

**Comment by Ivan-Khar** [comment]
Latest version of this mod is 1.3.2, but if you need to use 1.3.11 you can get older version of the wiki

**Comment by Ivan-Khar** [comment]
https://github.com/AquUpd/CaracalMob/wiki/Nametags/bcbcb359ebd424b16487186f3189490e605d2b6b

**Comment by jadeitemods** [comment]
Whoopsie. I guess the information on Curseforge was out of date?

**Reply by Ivan-Khar** [reply]
Information on curseforge for latest version of the mod(1.3.2)
This wiki page is for older versions of the mod(1.3.11 and lower)
https://github.com/AquUpd/CaracalMob/wiki/Nametags/bcbcb359ebd424b16487186f3189490e605d2b6b

**Comment by jadeitemods** [comment]
Was the scuba mask one removed?

**Reply by Ivan-Khar** [reply]
1.3.11 version is older than 1.3.2
Just get 1.3.2 and you should get all new masks

**Comment by jadeitemods** [comment]
Final question, sorry. Is the "Floppa" name supposed to give it the top
hat? It's not doing anything.

**Reply by Ivan-Khar** [reply]
It's should give hat

**Comment by jadeitemods** [comment]
[image: image.png]

**Comment by jadeitemods** [comment]
The other ones all work.

**Comment by jadeitemods** [comment]
That's just it. I have swapped over to 1.3.2.

**Comment by Ivan-Khar** [comment, marked as answer]
As i said earlier. Just get 1.3.2 version and after that you should easily use new naming system and get new scuba gear mask

**Comment by jadeitemods** [comment]
As I said earlier, I am currently, as we speak, running 1.3.2. The new
naming system functions for all names shown here, on the link you sent me.
[image: image.png]
This is a screenshot. The only one that doesn't work is the tophat,
"Floppa". The scuba one is not on the new list. Is it still "Water"?

**Reply by Ivan-Khar** [reply]
Use wiki from curseforge, wiki page that I sent to you was for 1.3.11

**Comment by jadeitemods** [comment]
Edit: the scuba one does work. The tophat one does not.

**Comment by jadeitemods** [comment]
Okay. I think we both were confused at what the other was saying, but I
figured it out. Thanks for the help!
